### ISSUE TITLE
levitus SST and SSS have different lon0 values

### ISSUE THREAD

**Issue by dankelley** [dankelley/ocedata#7]
Gosh, this is embarrassing!
<img width="601" alt="screen shot 2017-07-04 at 9 27 12 am" src="https://user-images.githubusercontent.com/99469/27830100-2e71ed7e-609b-11e7-9bca-2c147346f77b.png">

**Comment by dankelley** [comment]
Hm, this is a weird one. The code to create this, given below, seems perfectly reasonable. I'll try re-downloading the data files ...
```R
library(oce)
library(ncdf4)
#con <- nc_open("temperature_annual_1deg.nc")
get <- function(f, item) {
    res <- ncvar_get(f, item)
    res <- if (1 == length(dim(res))) as.vector(res) else array(res, dim=dim(res))
    res[!is.finite(res)] <- NA
    res
}

con <- nc_open("/data/oar/levitus/woa13_decav_t00_01.nc")
longitude <- get(con, "lon")
latitude <- get(con, "lat")
SST <- get(con, "t_an")[,,1]
SST[SST > 100] <- NA

con <- nc_open("/data/oar/levitus/woa13_decav_s00_01.nc")
con <- nc_open("salinity_annual_1deg.nc")
SSS <- get(con, "s_an")[,,1]
SSS[SSS > 100] <- NA

## Shift longitude
lon2 <- ifelse(longitude > 180, longitude - 360, longitude)
i  <- order(lon2)
longitude <- lon2[i]
SST <- SST[i, ]
SSS <- SSS[i, ]

## Save and compress
levitus <- list(longitude=longitude, latitude=latitude, SSS=SSS, SST=SST)
save(levitus, file='levitus.rda')
require(tools)
tools::resaveRdaFiles("levitus.rda")

## Check with a graph

par(mfrow=c(2,1))
imagep(levitus$longitude, levitus$latitude, levitus$SST)
imagep(levitus$longitude, levitus$latitude, levitus$SSS)
```

**Comment by dankelley** [comment]
Fixed in `master` branch, commit 37f6a0f4193423c8ef7fc63d5f9208e7a8bcf6e3

**Comment by dankelley** [comment]
ps. the problem was here
```R
con <- nc_open("/data/oar/levitus/woa13_decav_s00_01.nc")
con <- nc_open("salinity_annual_1deg.nc")
```
(the 2nd line was giving a different dataset -- silly me). I updated the data anyway -- still the 2013 dataset, but a clean download, just in case.
